### DISCUSSION TITLE
QR for controlling usb connection to mtp hero8 black

### DISCUSSION THREAD

**Discussion by rdelav** [gopro/labs#690] [Q&A]
hi, my screen is broken and i have been using qr codes since I saw I can control it. Is there a way that I can control the usb connection to MTP so I can finally download my images to my win10. Currently it just charges it, so I saw this video https://www.youtube.com/watch?v=GZJeuX8tAf4 and found out that control doesnt exist in the pref. that are already there. I cant connect to the wifi right now, sice I have wifi just to one device, plus I dont have a slot for reading a micro sd.

If it exists, can I get to see the qr? please, thank you for your time.

**Comment by dnewman-gpsw** [comment]
That particular command doesn't exist via QR code.  A microSD to USB adapter is your best option. They often come free with SD cards, or you can buy them for under $10.

**Reply by dnewman-gpsw** [reply]
E.g. https://a.co/d/9eDNpcG

**Reply by rdelav** [reply]
i am currently not been able to buy one (was in the middle of the sea in vacations) buut i found out my redmi 7 has one on it. so i managed to download all via my phone! thanks again for your time

**Reply by rdelav** [reply]
but would be nice, if those other pref could be found via qr for the people that lost their back screen like

**Reply by dnewman-gpsw** [reply]
It is not an issue on later models, as they can automatically detect and USB protocols.

**Reply by rdelav** [reply]
:( is it so difficult to make the lines for it or the qr? just wondering, like maybe to try to dive into it. Maybe we are so few with the old cameras? I wonder why it has this problem in older models...

**Reply by dnewman-gpsw** [reply]
Yes, too difficult to add corrnercase camera controls in past releases. In the almost four years since HERO8 Labs, you are the first ask for this control. Labs never intended for feature completeness in QR codes, rather it is playing field to experiment with new features.

I think the reason this issue only appears on H8, It was the first introduced webcam and hadn't resolved how to manage the webcam and MTP protocols at the same time. Therefore requiring the on-screen user selection.

**Reply by rdelav** [reply]
wow, since this one is not actually mine but my brother i wasnt aware it could be a webcam. I guess... i cant use it if i cant access the controls right? :(
The qr actually saved me, because the touch camera stop working and couldnt even control it anymore. anyway. thanks again
